ph5/utilities/ph5validate.py

```diff
@@ -863,17 +866,14 @@ def main():
         args = get_args()
         ph5API_object = ph5api.PH5(path=args.ph5path, nickname=args.nickname)
         ph5validate = PH5Validate(ph5API_object,
-                                  args.ph5path,
-                                  args.level.upper(),
-                                  args.outfile)
+                                  args.ph5path)
         validation_blocks = []
         validation_blocks.extend(ph5validate.check_experiment_t())
         validation_blocks.extend(ph5validate.check_array_t())
         validation_blocks.extend(ph5validate.check_event_t())
         with open(args.outfile, "w") as log_file:
             for vb in validation_blocks:
-                vb.write_to_log(log_file,
-                                args.level)
+                vb.write_to_log(log_file, args.level.upper())
```

Comment:
This should use the logger

Reply:
[write_to_log() ](https://github.com/PIC-IRIS/PH5/blob/f4e8f1b4ebd79ef3c3c65de7d797e4062023cb26/ph5/utilities/ph5validate.py#L39)will write into ph5_validate.log in a special format for each station, whole event_t, response_t or experiment_t check with the header following with info/warning/error depending on the level selected.
Ex:

```
-=-=-=-=-=-=-=-=-
Experiment_t
1 error, 0 warning, 0 info
-=-=-=-=-=-=-=-=-
ERROR: Experiment_t does not exist. run experiment_t_gen to create table
-=-=-=-=-=-=-=-=-
Station 1111 Channel 1
2 error, 0 warning, 0 info
-=-=-=-=-=-=-=-=-
ERROR: Channel longitude 327519.7 not in range [-180,180]
ERROR: Channel latitude 3773952.7 not in range [-90,90]
-=-=-=-=-=-=-=-=-
Station 1111 Channel 1
2 error, 0 warning, 0 info
-=-=-=-=-=-=-=-=-
ERROR: Channel longitude 327519.7 not in range [-180,180]
ERROR: Channel latitude 3773952.7 not in range [-90,90]
```

I can remove the format for the file logger and still use write_to_log() function but using logger to write instead of opening the ph5_validate.log file to write into. However, the new way will write every info, warning, error to both screen and file; while with the old way I can log important info like which table are being checked or serious error to screen, user will read in the log file for all info, warning, error. DG is already used to the old way.   

Example of log info on screen:

```
[2020-08-12 16:40:50,103] - ph5.utilities.ph5validate - INFO: Validating Experiment_t
[2020-08-12 16:40:50,104] - ph5.utilities.ph5validate - INFO: Validating Array_t
[2020-08-12 16:40:50,418] - ph5.utilities.ph5validate - INFO: Validating Event_t
[2020-08-12 16:40:50,419] - ph5.utilities.ph5validate - WARNING: Event_t table not found. Did this experiment have shots???

Warnings, Errors and suggestions written to logfile: ph5_validate.log
```

If I change to use logger for everything, all will be printed to screen and make it harder to see important information. Please tell me if you want to proceed it that way.

Reply:
This isn't good, but leave it.